package.json

```diff
@@ -15,6 +15,9 @@
     "dev": "tsc-watch --onSuccess \"npm run webpack-run\"",
     "precommit": "npm run build && npm run test && npm run lint"
   },
+  "pre-commit": [
+    "precommit"
```

Comment:
you probably want to replace this with:
"test"
"lint"

Reply:
It runs ```npm run precommit``` 

Which is equivalent to ``` npm run build && npm run test && npm run lint```

https://github.com/flamecoals/turnato/blob/5fba8c5824e437c349080a6f3d69d77516645001/package.json#L16

![Spongebob Meme: MAGIC!](http://www.quickmeme.com/img/1e/1e316e8b30384c27aa347055973c419a658e6673e630d193304482261ef5de28.jpg)